test_container/tests/test/java/test_hadoop_etl_udfs.py

```diff
@@ -115,7 +123,8 @@ def add_containers_to_hosts_file_in_docker_db(self, env):
     def upload_hadoop_etl_udf_jar_to_bucket_fs(self, env):
         docker_db_container = env.get_docker_db_container()
         docker_db_ip = env.get_ip_address_of_container(docker_db_container)
-        upload_url = "http://{docker_db_ip}:6583/myudfs/hadoop-etl-udfs-v0.0.1-apache-2.8.5-3.0.0.jar".format(
+        bucketfs_port = bucket_fs_port_from_environment_info_file()
+        upload_url = f"http://{docker_db_ip}:{bucketfs_port}/myudfs/hadoop-etl-udfs-v0.0.1-apache-2.8.5-3.0.0.jar".format(
             docker_db_ip=docker_db_ip)
         username = "w"
         password = "write"  # TOOD hardcoded
```

Comment:
When we replace the copies, we should also replace the hard coded credentials and the docker_db_ip